client/components/Scenario/ContentSlide.jsx

```diff
@@ -62,6 +69,11 @@ class ContentSlide extends React.Component {
         const { name, value } = data;
         const { pending, required } = this.state;
 
+        if (!this.props.run) {
+            alert('Slides cannot accept responses in Preview');
```

Comment:
Likewise, not sure we need an alert here.

Reply:
I'd be pretty annoyed if I typed stuff into the prompts while in preview mode, hit "submit" and nothing happened.

Reply:
Removed. But I want to come up with some kind of feedback